roles/kubernetes/preinstall/defaults/main.yml

```diff
@@ -98,6 +98,12 @@ ntp_servers:
 ntp_restrict:
   - "127.0.0.1"
   - "::1"
+# Specify whether to filter interfaces
+ntp_filter_interface: false
+# Specify the interfaces
+# Only takes effect when ntp_filter_interface is true
+ntp_interfaces:
+  - ens3
```

Comment:
It's risky. `ens3` is an exact NIC name and is not applicable in most scenarios. I prefer to comment them out and suggest that when `ntp_filter_interface` is true, we should update the value of `ntp_interfaces` to match the node's NIC name.

But, do you think this still works for scenarios with inconsistent NIC names per node?

Reply:
Sure! I will comment out the `ntp_interfaces` section and add your suggestion as a comment.

When having inconsistent NIC names per node, One can have all interfaces from all nodes listed under this list of interfaces and if the interface does not exist on a node, I did a quick test and NTP but simply just listens on the set of interfaces that do exist.

```
interface listen ens3
interface listen test
```
The ntp server logs looked something like this when I restarted the service 

```
systemd[1]: Started Network Time Service.
ntpd[252818]: proto: precision = 0.050 usec (-24)
ntpd[252818]: restrict: 'monitor' cannot be disabled while 'limited' is enabled
ntpd[252818]: Listen normally on 0 lo 127.0.0.1:123
ntpd[252818]: Listen normally on 1 ens3 192.121.208.150:123
ntpd[252818]: Listen normally on 2 lo [::1]:123
ntpd[252818]: Listen normally on 3 ens3 [2a0a:bcc0:40::632]:123
ntpd[252818]: Listen normally on 4 ens3 [fe80::f816:3eff:fe9f:4fd3%2]:123
ntpd[252818]: Listening on routing socket on fd #21 for interface updates
```

Reply:
Hi @Pavan-Gunda , It looks like the `ntp_interfaces` section is still not commented out. Is that right?

Reply:
Hi! 

I commented `ntp_interfaces` section out now. :)